app/assets/stylesheets/admin/_form.scss

```diff
@@ -0,0 +1,7 @@
+.form .event_description textarea {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `event_description` should be written in lowercase with hyphens